sql/set.go

```diff
@@ -94,7 +94,7 @@ func (p *planner) SetTimeZone(n *parser.SetTimeZone) (planNode, *roachpb.Error)
 			location = "UTC"
 		}
 		if _, err := time.LoadLocation(location); err != nil {
-			return nil, roachpb.NewError(err)
+			return nil, roachpb.NewUErrorf("cannot find %s: %v", location, err)
```

Comment:
%s for error. should we use this opportunity to give a better error message? pg gives `ERROR:  invalid value for parameter "TimeZone": "blabla"`